tests/riscv64/lit.cfg.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+import tempfile
+
+import lit.formats
+
+config.name = "RISC-V tests"
+config.test_format = lit.formats.ShTest(execute_external=True)
+
+config.suffixes = [".run"]
+
+BUILD_RISCV_DIR = os.path.abspath(os.environ["BUILD_RISCV_DIR"])
+
+config.environment["BUILD_RISCV_DIR"] = BUILD_RISCV_DIR
+
+test_cmd = [
+    os.environ["QEMU_RV64_BIN"],
+    "-cpu",
+    "rv64,x-v=true,x-k=true,vlen=256,elen=64,vext_spec=v1.0",
```

Comment:
```suggestion
    "rv64,x-v=true,x-k=true,vlen=512,elen=64,vext_spec=v1.0",
```

Based on the recent exploration by @dcaballe